pages/common/vboxmanage-movevm.md

```diff
@@ -0,0 +1,12 @@
+# vboxmanage-movevm
+
+> Move a virtual machine (VM) to a new location on the host system.
+> More information: <https://www.virtualbox.org/manual/ch08.html#vboxmanage-movevm>.
+
+- Move the specified VM:
+
+`VBoxManage movevm {{vm_name}}`
```

Comment:
Where does it get moved to?

Reply:
Ah my bad will change it!

Reply:
@gutjuri do you know why jiamvy is making the changes and not my account jimsjam?

Reply:
@JimsJam Can you check `git --config user.name` and `git --config email` locally to see if they match your GitHub name & email address?

Reply:
@kbdharun Thank you so much for the help. The e-mail was actually a different one.